tests/validation/createcluster.go

```diff
@@ -0,0 +1,72 @@
+package validation
+
+import (
+	. "./testutils"
```

Comment:
Go frowns upon relative path imports. Can you reference this package from it's proper GOPATH import?